azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/ConfigMojo.java

```diff
@@ -184,17 +175,20 @@ private WebAppConfiguration getWebAppConfiguration(WebAppConfiguration configura
             NOT_EMPTY_REGEX, null, null);
 
         final String defaultRegion = configuration.region != null ? configuration.region.name() :
```

Comment:
add EUROPE_WEST as a constant default value in configuration, provide a getRegion method returning with the failback-to-default code.